src/main/java/de/tum/in/www1/artemis/repository/ProgrammingExerciseRepository.java

```diff
@@ -54,13 +54,19 @@
             + "and (spr.id = (select max(id) from sp.results) or spr.id = null)")
     Optional<ProgrammingExercise> findWithTemplateAndSolutionParticipationLatestResultById(@Param("exerciseId") Long exerciseId);
 
-    @Query("select distinct pe from ProgrammingExercise as pe left join fetch pe.studentParticipations")
+    @Query("select distinct pe from ProgrammingExercise pe left join fetch pe.studentParticipations")
```

Comment:
```suggestion
    @Query("""
    select distinct pe 
    from ProgrammingExercise pe left join fetch pe.studentParticipations
    """)
```

Reply:
Most queries in this file are one-liners. Imo we should either change all for consistency or leave this as is.